docs/changelog.rst

```diff
@@ -19,6 +19,7 @@ Added
 - I/O:
     - SVG drawer for SiDB layouts
 - Experiments:
+    - Added information about how many layouts remain after each of QuickCell's pruning steps.
```

Comment:
v0.6.6 has already been released. Please add the change under a new `Unreleased` header.

Reply:
True! This was caused by the github merge. Should be fixed now.